### DISCUSSION TITLE
Possible virus repo?

### DISCUSSION THREAD

**Discussion by thenintendofreak** [community/community#82768] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello I downloaded a program called bnrtool and i believe it was a virus. Can someone make sure for me and if so take it down? Thank you

https://github.com/xchellx/bnrtool

**Comment by ghostinhershell** [comment]
Hi @thenintendofreak, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. You can learn more about reporting behavior and content that violates community guidelines and terms in our Docs here: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam.

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the link above.

Thank you!

**Reply by thenintendofreak** [reply]
I did it's been like 5 days?!!
